install_dbx.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Install DB Connect dependencies
+  hosts: heavyforwarder
+  remote_user: root
+  tasks:
+   - name: install packages for dbconnect addon
+     apt: name={{item}} state=present update_cache=yes
+     with_items:
+       - openjdk-8-jre
+
+   - name: set up java home envvar
+     lineinfile:
+       - path: /etc/environment
+       - line: 'JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"'
+       - state: present
```

Comment:
Is this the first and only addon that needs the JRE? Will this clobber any existing `JAVA_HOME`s?

Reply:
The first we've run across, yes.  I'm not sure what the future holds though..